docs/general/account-billing/change-email-address-or-password-for-your-beaver-builder-account.md

```diff
@@ -4,5 +4,11 @@ title: Change email address or password for your Beaver Builder account
 sidebar_label: Change email address or password
 ---
 
-You can change your email address or password for your Beaver Builder account
+### Email Address
+
+You can change your email address for your Beaver Builder account on the edit account page. However, this will not update your billing address. Therefore, if you need to change your email address, please [contact us](https://www.wpbeaverbuilder.com/contact/) and our billing team will handle your request.
```

Comment:
You can change your email address for your Beaver Builder account on the [Edit Account](https://www.wpbeaverbuilder.com/my-account/edit-account/) page but updating it there does not update your billing email. If you need to change your email address, [contact us](https://www.wpbeaverbuilder.com/contact/) and our billing team will handle your request.